WorkbenchProject/src/main/java/edu/uwb/braingrid/workbench/ui/ScriptSpecificationDialog.java

```diff
@@ -509,10 +507,10 @@ public ScriptSpecificationDialog(boolean modal, SimulationSpecification simSpec)
         String userName = simSpec.getUsername();
         String folder = simSpec.getSimulatorFolder();
         String codeLocation = simSpec.getCodeLocation();
+        codeLocation = codeLocation == null || codeLocation.isEmpty()
+                ? DEFAULT_REPO_URI : codeLocation;
```

Comment:
OK, two comments:

1. It's probably best to use some parentheses on complex expressions, rather than rely on operator precedence, for the sake of code readability. For example, I'd write `(codeLocation == null || codeLocation.isEmpty())`.
2. Given that the "else" part of the trinary assignment leaves `codeLocation` unchanged, is this really simpler than:
```C++
if (codeLocation == null || codeLocation.isEmpty()) codeLocation = DEFAULT_REPO_URI;
```

Reply:
To add some context, my only change to this line was replacing `getDefaultCodeLocation()` with `DEFAULT_REPO_URI` (since `getDefaultCodeLocation()` was a private helper function that did nothing but return a constant). However, I agree that using parentheses for the complex expression is a good practice here.

A single-line if statement does not fit the style guide, so the alternative would be:
```java
if (codeLocation == null || codeLocation.isEmpty()) {
    codeLocation = DEFAULT_REPO_URI;
}
```

I think using the if statement version is a slight improvement, but either way is fine with me.